data/data/bootstrap/files/usr/local/bin/release-image-download.sh.template

```diff
@@ -17,3 +17,16 @@ do
     echo "Pull failed. Retrying $RELEASE_IMAGE..."
 done
 
+# Sanity check the image metadata to see if the arches match
+image_arch=$(podman inspect $RELEASE_IMAGE --format {{ "{{.Architecture}}" }})
+host_arch=$(uname -m)
+case $host_arch in
```

Comment:
Are the `ppc64le` and `s390x` architectures not included here because they do not need any mangling?

Reply:
correct. `x86_64` and `aarch64` map to `amd64` and `arm64` resp. it is a cause of confusion in many places.